UPGRADE-1.x.md

```diff
@@ -1,3 +1,12 @@
+UPGRADE FROM 1.2.6 to 1.2.7
```

Comment:
I think you should put this under `UPGRADE 1.x`. Move that under a fixed version is part of the release process.